aim/sdk/run.py

```diff
@@ -236,7 +244,8 @@ def __init__(self, run_hash: Optional[str] = None, *,
                  repo: Optional[Union[str, 'Repo']] = None,
                  read_only: bool = False,
                  experiment: Optional[str] = None,
-                 system_tracking_interval: Optional[int] = DEFAULT_SYSTEM_TRACKING_INT):
+                 system_tracking_interval: Optional[int] = DEFAULT_SYSTEM_TRACKING_INT,
+                 log_system_params: Optional[bool] = True):
```

Comment:
lets make `False` by default at least for now. Without UI changes this will blow the Run/Metric explorer tables.